tests/accuracy/test_fp32_accuracy.cpp

```diff
@@ -50,6 +50,6 @@ TEST_P(AccuracyTests, FP16AccuracyIsClose) {
 }
 
 INSTANTIATE_TEST_SUITE_P(
-    FP16AccuracyIsCloseSuite,
+    FP32AccuracyIsCloseSuite,
     AccuracyTests,
-    testing::Values("tests/accuracy/vgg16_cifar10.jit.pt"));
+    testing::Values("tests/accuracy/resnet50_scripted.jit.pt"));
```

Comment:
can we keep this as VGG?

Reply:
Where can I get vgg16_cifar10.jit.pt ?
I couldn't find it so I used resnet50_scripted.jit.pt from hub.py scripts

Reply:
done